syntax="proto3";

package rpc_Proto;


message RpcHeader
{
    bytes service_name=1;
    bytes method_name=2;
    uint32 args_size=3;
}
